Solutions/Vectra/Data Connectors/Connector_VectraAI_Stream.json

```diff
@@ -0,0 +1,162 @@
+            "query": "VectraStream \n| wheremetadata_type == \"metadat_dns\" \n| summarize count() by type_name" 
+        },
+        {
+            "description" : "Top 10 of query to non existing domain",
+            "query": "VectraStream \n| wheremetadata_type == \"metadat_dns\" \n| where rcode_name == \"NXDomain\"\n| summarize Count=count() by tostring(query)\n| order by Count desc\n| limit 10"
+        },
+        {
+            "description" : "Host and Web sites using non-ephemeral Diffie-Hellman key exchange",
+            "query": "VectraStream \n| wheremetadata_type == \"metadat_dns\" \n| where cipher contains \"TLS_RSA\"\n| distinct orig_hostname, id_orig_h, id_resp_h, server_name, cipher\n| project orig_hostname, id_orig_h, id_resp_h, server_name, cipher" 
+        }
+    ],
+    "dataTypes": [ 
+        { 
+            "name": "VectraStream_CL",
+            "lastDataReceivedQuery": "\nVectraStream\n |  summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)" 
+        } 
+    ], 
+    "connectivityCriterias": [ 
+        { 
+            "type": "IsConnectedQuery", 
+            "value": [ 
+                "\nVectraStream\n| summarize LastLogReceived = max(TimeGenerated)\n| project IsConnected = LastLogReceived > ago(30d)" 
+            ] 
+        } 
+    ],
+    "availability": { 
+        "status": 1,
+        "isPreview": true
+    },
+    "permissions": {
+        "resourceProvider": [
+            {
+                "provider": "Microsoft.OperationalInsights/workspaces",
+                "permissionsDisplayText": "read and write permissions are required.",
+                "providerDisplayName": "Workspace",
+                "scope": "Workspace",
+                "requiredPermissions": {
+                    "write": true,
+                    "read": true,
+                    "delete": true
+                }
+            },
+            {
+                "provider": "Microsoft.OperationalInsights/workspaces/sharedKeys",
+                "permissionsDisplayText": "read permissions to shared keys for the workspace are required. [See the documentation to learn more about workspace keys](https://docs.microsoft.com/azure/azure-monitor/platform/agent-windows#obtain-workspace-id-and-key).",
+                "providerDisplayName": "Keys",
+                "scope": "Workspace",
+                "requiredPermissions": {
+                    "action": true
+                }
+            }
+        ],
+        "customs": [
+            {
+                "name": "Vectra AI Brain",
+                "description": "must be configured to export Stream metadata in JSON"
+            }
+        ]
+    },
+    "instructionSteps": [
+        { 
+            "title": "", 
+            "description": ">**NOTE:** This data connector depends on a parser based on a Kusto Function to work as expected **VectraStream** which is deployed with the Azure Sentinel Solution.",
```

Comment:
Parser link should be the part of this note. Please add the parser link in this format https://aka.ms/sentinel-Parser-AIVectraStream.